homeassistant/components/climate/const.py

```diff
@@ -67,6 +67,7 @@
 FAN_MIDDLE = "middle"
 FAN_FOCUS = "focus"
 FAN_DIFFUSE = "diffuse"
+FAN_WIND = "wind"
```

Comment:
Changes to an entity model like climate, must first be approved in an issue in our architecture repo:
https://developers.home-assistant.io/docs/en/entity_index.html#changing-the-entity-model